azurerm/resource_arm_app_service_certificate_order.go

```diff
@@ -0,0 +1,488 @@
+		d.Set("intermediate", flattenArmCertificateOrderCertificateDetails(props.Intermediate))
+	}
+
+	return tags.FlattenAndSet(d, resp.Tags)
+}
+
+func resourceArmAppServiceCertificateOrderDelete(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient).web.CertificatesOrderClient
+
+	id, err := azure.ParseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resourceGroup := id.ResourceGroup
+	name := id.Path["certificateOrders"]
+
+	log.Printf("[DEBUG] Deleting App Service Certificate Order %q (Resource Group %q)", name, resourceGroup)
+
+	ctx := meta.(*ArmClient).StopContext
+	resp, err := client.Delete(ctx, resourceGroup, name)
+	if err != nil {
+		if !utils.ResponseWasNotFound(resp) {
+			return fmt.Errorf("Error deleting App Service Certificate Order %q (Resource Group %q): %s)", name, resourceGroup, err)
+		}
+	}
+
+	return nil
+}
+
+func flattenArmCertificateOrderCertificate(input map[string]*web.AppServiceCertificate) []interface{} {
+	results := make([]interface{}, 0)
+
+	for k, v := range input {
+		result := make(map[string]interface{})
+
+		result["certificate_name"] = k
+
+		if keyVaultID := v.KeyVaultID; keyVaultID != nil {
+			result["key_vault_id"] = *keyVaultID
+		}
+		if keyVaultSecretName := v.KeyVaultSecretName; keyVaultSecretName != nil {
+			result["key_vault_secret_name"] = *keyVaultSecretName
+		}
+		result["provisioning_state"] = string(v.ProvisioningState)
+
+		results = append(results, result)
+	}
+
+	return results
+}
+
+func flattenArmCertificateOrderCertificateDetails(input *web.CertificateDetails) []interface{} {
+	results := make([]interface{}, 0)
+
+	if input == nil {
+		return results
+	}
+
+	result := make(map[string]interface{})
+
+	if version := input.Version; version != nil {
+		result["version"] = version
+	}
+
+	if serialNumber := input.SerialNumber; serialNumber != nil {
+		result["serial_number"] = serialNumber
+	}
+
+	if thumbprint := input.Thumbprint; thumbprint != nil {
+		result["thumbprint"] = thumbprint
+	}
+
+	if subject := input.Subject; subject != nil {
+		result["subject"] = subject
+	}
+
+	if notBefore := input.NotBefore; notBefore != nil {
+		result["not_before"] = notBefore.Format(time.RFC3339)
+	}
+
+	if notAfter := input.NotAfter; notAfter != nil {
+		result["not_after"] = notAfter.Format(time.RFC3339)
+	}
+
+	if signatureAlgorithm := input.SignatureAlgorithm; signatureAlgorithm != nil {
+		result["signature_algorithm"] = signatureAlgorithm
+	}
+
+	if issuer := input.Issuer; issuer != nil {
+		result["issuer"] = issuer
+	}
+
+	if rawData := input.RawData; rawData != nil {
+		result["raw_data"] = rawData
+	}
+
+	results = append(results, result)
+
+	return results
+}
+
+func appServiceCertificateDetailsSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"version": {
+			Type:     schema.TypeInt,
+			Computed: true,
+		},
+
+		"serial_number": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+
+		"thumbprint": {
```

Comment:
Many of the attributes here can be removed but I think `thumbprint` is useful. Also, how many Certificates can this resource hold? If it's just one, we can just have a thumbprint attribute and remove this TypeList entirely

Reply:
Remove the computed attributes seem useless.